resources/ru/cemu.po

```diff
@@ -256,11 +256,11 @@ msgstr "&Точки останова"
 # Shold ampersand be there?
```

Comment:
The ampersand is positioned just before the letter that is used for the menu shortcut, feel free to move it in a more appropriate position according to your language.
(PS: remove this line the next time you edit this file)